process/process_linux.go

```diff
@@ -220,8 +220,8 @@ func (p *Process) NumCtxSwitches() (*NumCtxSwitchesStat, error) {
 
 // NumFDs returns the number of File Descriptors used by the process.
 func (p *Process) NumFDs() (int32, error) {
-	numFds, _, err := p.fillFromfd()
-	return numFds, err
+	_, fds, err := p.fillFromfdList()
```

Comment:
can we just push the stuff in `fillFromfdList` into this function since it's not used anywhere else?